cmd/gaia/app/app.go

```diff
@@ -242,8 +246,11 @@ func (app *GaiaApp) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.R
 	validatorUpdates, endBlockerTags := staking.EndBlocker(ctx, app.stakingKeeper)
 	tags = append(tags, endBlockerTags...)
 
-	if app.assertInvariantsBlockly {
-		app.assertRuntimeInvariants()
+	if app.invariantsChecksFrequency > 0 {
+		if ctx.BlockHeight() == int64(app.nextInvariantsCheck) || app.nextInvariantsCheck == 0 {
```

Comment:
`ctx.BlockHeight() % app.invariantsChecksFrequency == 0` is simpler

Reply:
> The thing you call "frequency" is a "period" between invariant checks - a "frequency" would be its inverse. 

You're absolutely right, thanks!

> ctx.BlockHeight() % app.invariantsChecksFrequency == 0 is simpler

Nice one! Will do :+1: